R/TealAppDriver.R

```diff
@@ -388,6 +388,15 @@ TealAppDriver <- R6::R6Class( # nolint
       invisible(self)
     },
     #' @description
+    #' Wrapper around `get_html` that passes the output directly to `rvest::read_html`.
+    #'
+    #' @param selector `(character(1))` passed to `get_html`.
+    #'
+    #' @return An XML document.
+    read_html = function(selector) {
```

Comment:
I'm not sure about the name as it's a new `read_*` "namespace"

Maybe rename it to `get_html_rvest` or create a `get_attr` function that works just as other `get_text`, etc...

`{rvest}` seems to be used only to get the attribute value.

Reply:
thanks, will consider!

Reply:
Hey, renamed to bdd19123098faea581b691c78f630ceb81023a10 `get_html_rvest`